### PR TITLE
change location of compress

### PR DESCRIPTION
**Pull request by samwaseda**
Make it possible to take a look at the output of running jobs without making changes to output files.

### CODE REVIEW

File: pyiron_atomistics/sphinx/base.py [hunk 1]
```diff
@@ -1940,7 +1940,7 @@ def collect_spins_dat(self, file_name="spins.dat", cwd=None):
             return None
         spins = np.loadtxt(posixpath.join(cwd, file_name))
         self._parse_dict["atom_scf_spins"] = self.splitter(
-            np.array([ss[self._job.id_spx_to_pyi] for ss in spins[:, 1:]]),
+            [ss[self._job.id_spx_to_pyi] for ss in spins[:, 1:]],
```
Comment (jan-janssen) [inline pyiron_atomistics/sphinx/base.py]: Why do you remove the `numpy.array()` ?

Comment (samwaseda) [reply pyiron_atomistics/sphinx/base.py]: Oh sorry it was not meant to be part of this PR.

File: pyiron_atomistics/sphinx/interactive.py [hunk 2]
```diff
@@ -213,8 +213,8 @@ def _output_interactive_to_generic(self):
                                     "interactive/atom_spin_constraints"
                                 ]
 
-    def collect_output(self, force_update=False):
-        super(SphinxInteractive, self).collect_output(force_update=force_update)
+    def collect_output(self, force_update=False, compress=True):
+        super(SphinxInteractive, self).collect_output(force_update=force_update, compress=compress)
```
Comment (jan-janssen) [inline pyiron_atomistics/sphinx/interactive.py]:
```suggestion
    def collect_output(self, force_update=False, compress_files=True):
        super(SphinxInteractive, self).collect_output(force_update=force_update, compress_files=compress_files)
```

File: pyiron_atomistics/sphinx/base.py (hunk 1 above)
Comment (jan-janssen) [reply pyiron_atomistics/sphinx/base.py]:
```suggestion
            np.array([ss[self._job.id_spx_to_pyi] for ss in spins[:, 1:]]),
```

**Comment by samwaseda** [comment]
I wonder whether this makes maybe also sense for VASP.

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Comment by jan-janssen** [comment]
> I wonder whether this makes maybe also sense for VASP.

@sudarsan-surendralal

**Pull request merged**